aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -329,12 +328,7 @@ impl AptosVM {
         // Also, it is worth mentioning that current VM error handling is
         // rather ugly and has a lot of legacy code. This makes proper error
         // handling quite challenging.
-        let delta_write_set = user_txn_change_set_ext
-            .delta_change_set()
-            .clone()
-            .try_into_write_set(resolver)?;
-        let delta_state_view = DeltaStateView::new(&storage_with_changes, &delta_write_set);
-        let resolver = delta_state_view.as_move_resolver();
+        let resolver = storage_with_changes.as_move_resolver();
```

Comment:
could combine the definition of storage_with_changes in the same line, I think that makes sense.

Reply:
Done.

Reply:
oh, makes sense, I remember being able to have a resolver that took ownership but can only find the one now that holds a reference, so it needs a binding / owner.
Let's leave it as is then.